Code/Editor/EditorViewportSettings.cpp

```diff
@@ -61,6 +62,7 @@ namespace SandboxEditor
     constexpr AZStd::string_view CameraDefaultStartingYaw = "/Amazon/Preferences/Editor/Camera/DefaultStartingYaw";
     constexpr AZStd::string_view CameraNearPlaneDistanceSetting = "/Amazon/Preferences/Editor/Camera/NearPlaneDistance";
     constexpr AZStd::string_view CameraFarPlaneDistanceSetting = "/Amazon/Preferences/Editor/Camera/FarPlaneDistance";
+    constexpr AZStd::string_view CameraFovSetting = "/Amazon/Preferences/Editor/Camera/FovRadians";
```

Comment:
I think for users/customers, have the value stored in degrees is going to be much easier to understand than radians. So would it be possible to actually serialize the data in degrees, but then people can call `CameraDefaultFovRadians()` when needed to get it in radians.

So what I'm suggesting is...

```
constexpr AZStd::string_view CameraFovDegreesSetting = "/Amazon/Preferences/Editor/Camera/FovDegrees";
```

(and see my later comment below where we swap the implementation around just slightly)